tests/001-test-use-devs-to-create-thin-pool.sh

```diff
@@ -17,6 +17,7 @@ test_devs() {
   cat << EOF > /etc/sysconfig/docker-storage-setup
 DEVS="$devs"
 VG=$vg_name
+CONTAINER_THINPOOL=docker-pool
```

Comment:
So here are my thoughts.
- If user did not pass in inputfile/outputfile and CONTAINER_THINPOOL is not set, pool name will be dokcker-pool. 
- If user did not pass in inputfile/outputfile and CONTAINER_THINPOOL is set, pool name will be as specified by CONTAINER_THINPOOL.
- If user did specify inputfile/outputfile but no CONTAINER_THINPOOL, this should error out if we are about to create new thin pool.
- If user did specify inputfile/output but no CONTAINER_THINPOOL, this should fall back to docker-pool for the case of upgrade and reset. This will ensure that users who used inputfile/outputfile and still got pool name as docker-pool continue to work and are not broken.